internal/command/operator.go

```diff
@@ -298,6 +302,26 @@ Note: If --auto-registry-credentials and --registry-credentials-path are unset,
 			if stdout {
 				applier = kubernetes.NewStdOutApplier()
 			} else {
+				// before starting the application of the installation instance,
+				// we can check if the installation CRD is present
+				kubeCfg, err := kubernetes.NewConfig(kubeConfig)
+				if err != nil {
+					return err
+				}
+
+				installationClient, err := operator.NewInstallationClient(kubeCfg)
+				if err != nil {
+					return err
+				}
+
+				_, err = installationClient.Status(ctx)
+				switch {
+				case errors.Is(err, operator.ErrNoInstallationCRD):
+					return fmt.Errorf("no installation CRD found in cluster %q, have you run 'jsctl operator deploy'?", kubeCfg.Host)
```

Comment:
Nit: here, could we provide the full name of the CRD?

Reply:
fixed in 1ba9f9b3ce2df61089c6b5d56632b2335d5ca131